kong/plugins/zipkin/schema.lua

```diff
@@ -15,6 +40,7 @@ return {
           { traceid_byte_count = { type = "integer", required = true, default = 16, one_of = { 8, 16 } } },
           { header_type = { type = "string", required = true, default = "preserve",
                             one_of = { "preserve", "b3", "b3-single", "w3c" } } },
+          { static_tags = { type = "array", elements = static_tag } }
```

Comment:
I doesn't look like there's a way to enforce in the schema that the `static_tag` values' names are unique. The last one wins if there are multiple tags with the same name. Should we validate that the names are unique?

Reply:
This is now included (code and tests)